src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -472,7 +472,6 @@ private function sortByAlgorithm($indexes)
                 break;
             default:
                 throw new \LogicException(sprintf('Sort algorithm "%s" is not supported.', $this->configuration['sortAlgorithm']));
-                break;
```

Comment:
just a minor (I won't comment on all minors, let me know if you've a question about one of those)